src/main/java/com/ecore/roles/web/dto/UserDto.java

```diff
@@ -0,0 +1,57 @@
+package com.ecore.roles.web.dto;
+
+import com.ecore.roles.client.model.User;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode
+public class UserDto {
+
+    @JsonProperty
```

Comment:
Add all the JSON properties names in the JsonProperty annotations, in order to don't get a code refactoring in the fields names leading to a mismatch in the API clients.